recipes/coin-clp/all/conandata.yml

```diff
@@ -2,7 +2,13 @@ sources:
   "1.17.6":
     url: "https://github.com/coin-or/Clp/archive/releases/1.17.6.tar.gz"
     sha256: "afff465b1620cfcbb7b7c17b5d331d412039650ff471c4160c7eb24ae01284c9"
+  "1.17.7":
```

Comment:
Please, keep the newest version on top, it's easier for human reading. Not a blocker, but consider in the future.

Reply:
I've searched some common rule in recipes, didn't find one, now I have it ;)